### PR TITLE
Implement the combo detail UI

### LINKED ISSUE #23: Implement the combo detail UI
[Figma design](https://www.figma.com/design/d60yqkXZspwUMpdiyyFyZn/GlowUp-application?node-id=229-3329&node-type=frame&t=TRiibIHSioJCNBzb-0)

### PR DESCRIPTION
**Pull request by truongtdph40209**
## Related issues
Resolved #23 

## Summary
- Implement Combo Detail screen includes: 
   + Dark mode, light mode
- Update translation key 

### Steps:
1. Open `Index.js` and delete the code in render
2. **Paste** the following line:

```jsx
<ComboDetailTemplate/>
```
3. Run app

## Screenshots
| Light mode | Dark mode |
| --- | --- |
|image|image|
image|image|

### CODE REVIEW

File: components/molecules/BookNow.tsx
Comment (KMTus) [inline components/molecules/BookNow.tsx:1]: Do not name the file like this

File: components/molecules/BookNow.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { useColorScheme } from 'react-native'
+import { Text, type TextProps, XStack, YStack } from 'tamagui'
+
+import { PositiveButton } from '~/components/atoms/PositiveButton'
+import getColors from '~/constants/Colors'
+import { useAppFonts } from '~/hooks/useAppFonts'
+import useTranslation from '~/hooks/useTranslation'
+
```
Comment (KMTus) [inline components/molecules/BookNow.tsx:9]: We need to check the conditions if the customer does not select the staff, the booking button will be blurred into gray to carefully select the time and the staff.

File: components/molecules/InputForm.tsx
Comment (KMTus) [inline components/molecules/InputForm.tsx:1]: Why do you change the old component ?

File: components/molecules/OurServices.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import useTranslation from '~/hooks/useTranslation'
+
+const OurServices = (): React.ReactElement => {
+  const { t } = useTranslation()
+  return (
+    <YStack gap={30}>
+      <LabelTitle title={t('screens.details.ourServices')} />
+
+      <View>
+        <StepList />
```
Comment (KMTus) [inline components/molecules/OurServices.tsx:15]: I don't want a jaw to transmit him here, it's not good at all

File: components/molecules/InputForm.tsx
Comment (truongtdph40209) [reply components/molecules/InputForm.tsx:1]: i update color for Separator

File: components/templates/HomeTemplate.tsx [hunk 3]
```diff
@@ -95,8 +96,8 @@ const HomeTemplate = (): React.ReactElement => {
             view="20"
           />
         </>)}
-
```
Comment (NguyenVanViet2004) [inline components/templates/HomeTemplate.tsx:98]: Stylelist?

**Review by NguyenVanViet2004 — CHANGES_REQUESTED** [review]

File: components/templates/HomeTemplate.tsx (hunk 3 above)
Comment (truongtdph40209) [reply components/templates/HomeTemplate.tsx:98]: I don't quite understand what you mean. Explain further.

Comment (NguyenVanViet2004) [reply components/templates/HomeTemplate.tsx:98]: implement the style list into the home template below the combo list.

**Review by NguyenVanViet2004 — APPROVED** [review]

**Pull request merged**